keys/employees-keys/mike-banon-key-signed.asc

```diff
@@ -0,0 +1,176 @@
+-----BEGIN PGP MESSAGE-----

```

Comment:
@mikebdp2 I'm not sure what this file is, but it is not your public key signed by me. Public key should start with `-----BEGIN PGP PUBLIC KEY BLOCK-----` and was sent to you in encrypted email/

Reply:
@pietrushnic Now I successfully decipher your email with my private key, but don't see `-----BEGIN PGP PUBLIC KEY BLOCK-----` anywhere. Although it has a large set of ASCII bytes, when I try to save it as "mike-banon-key-signed.asc" and import to GPG, it says this key isn't valid.
```
--------------776273452647BE1B643C34A6
Content-Type: text/plain; charset=UTF-8;
 name="mike-banon-key-signed.asc"
Content-Transfer-Encoding: base64
Content-Disposition: attachment;
 filename="mike-banon-key-signed.asc"

LS0tLS1CRUdJTiBQR1AgUFVCTElDIEtFWSBCTE9DSy0tLS0tCgptUUlOQkdDYW9nVUJFQUMr
... a lot of bytes ...
UEdQIFBVQkxJQyBLRVkgQkxPQ0stLS0tLQo=
--------------776273452647BE1B643C34A6--
```

Reply:
@pietrushnic looks like I just needed to decode base64, seems to work now.